README.md

```diff
@@ -48,6 +48,30 @@ class AddAddressToUsers < ActiveRecord::Migration
 end
 ```
 
+## Configuration
+
+Additional config can optionally be passed either in the initializer or in the migration itself
+
+### As a block in the initializer
+
+```ruby
+PerconaMigrations.config do |c|
+  c.drop_old_table = false
+  c.max_load       = 'Threads_running=60'
+end
+```
+
+### As a hash argument to the `percona_alter_table` command
+
+```ruby
+def up
+  commands = columns.map { |name, type| "ADD COLUMN #{name} #{type}" }
+  options = { drop_old_table: false, max_load: 'Threads_running=60' }
```

Comment:
it might have been better calling these drop_old_table, so it matches percon doc. it's not a big deal so happy for you to leave it the way it's.

Reply:
I did think about that, but decided in the end to make use of the same symbols that can be passed by the config block.